services/src/apidoc.rs

```diff
@@ -1,89 +1,92 @@
-        handlers::workflows::load_workflow_handler,
+        handlers::session::session_handler,
+        handlers::session::session_project_handler,
+        handlers::session::session_view_handler,
+        handlers::workflows::dataset_from_workflow_handler,
         handlers::workflows::get_workflow_metadata_handler,
         handlers::workflows::get_workflow_provenance_handler,
-        handlers::workflows::dataset_from_workflow_handler,
+        handlers::workflows::load_workflow_handler,
+        handlers::workflows::register_workflow_handler,
     ),
     components(
-        SimpleSession,
+        schemas(
+            SimpleSession,
 
-        DataId,
-        DataProviderId,
-        DatasetId,
-        ExternalDataId,
-        IdResponse<WorkflowId>,
-        LayerId,
-        ProjectId,
-        SessionId,
-        UploadId,
-        WorkflowId,
+            DataId,
+            DataProviderId,
+            DatasetId,
+            ExternalDataId,
+            IdResponse<WorkflowId>,
+            LayerId,
+            ProjectId,
+            SessionId,
+            UploadId,
+            WorkflowId,
 
-        TimeInstance,
-        TimeInterval,
+            TimeInstance,
+            TimeInterval,
 
-        Coordinate2D,
-        BoundingBox2D,
-        SpatialPartition2D,
-        SpatialResolution,
-        SpatialReference,
-        SpatialReferenceOption,
-        SpatialReferenceAuthority,
-        Measurement,
-        ContinuousMeasurement,
-        ClassificationMeasurement,
-        STRectangle,
+            Coordinate2D,
+            BoundingBox2D,
+            SpatialPartition2D,
+            SpatialResolution,
+            SpatialReference,
+            SpatialReferenceOption,
+            SpatialReferenceAuthority,
+            Measurement,
+            ContinuousMeasurement,
+            ClassificationMeasurement,
+            STRectangle,
 
-        ProvenanceOutput,
-        Provenance,
+            ProvenanceOutput,
+            Provenance,
 
-        VectorDataType,
-        RasterDataType,
+            VectorDataType,
+            FeatureDataType,
+            RasterDataType,
 
-        VersionInfo,
+            VersionInfo,
 
-        Workflow,
-        TypedOperator,
-        TypedResultDescriptor,
-        PlotResultDescriptor,
-        RasterResultDescriptor,
-        VectorResultDescriptor,
-        RasterDatasetFromWorkflow,
-        RasterDatasetFromWorkflowResult,
-        RasterQueryRectangle,
-        VectorQueryRectangle,
-        PlotQueryRectangle,
+            Workflow,
+            TypedOperator,
+            TypedResultDescriptor,
+            PlotResultDescriptor,
+            RasterResultDescriptor,
+            VectorResultDescriptor,
+            VectorColumnInfo,
+            RasterDatasetFromWorkflow,
+            RasterDatasetFromWorkflowResult,
+            RasterQueryRectangle,
+            // VectorQueryRectangle,
+            // PlotQueryRectangle,
```

Comment:
I guess you forgot those.

Reply:
I commented them out because swaggerhub complains about these types being unused. But I already added the schema definition and we will use them in the future, so I commented them out instead of deleting. But I can also delete the lines.

Reply:
Does this result in the project not being builded?

Reply:
no the project builds fine but the resulting openapi.json generates warnings when I upload them to swaggerhub. Because you aren't supposed to include unreferenced components